solidity/dashboard/src/reducers/liquidity-rewards.js

```diff
@@ -0,0 +1,57 @@
+const wrappedTokenInitialData = {
+  shareOfPoolInPercent: 0,
+  reward: 0,
+  wrappedTokenBalance: 0,
+  lpBalance: 0,
+  isFetching: false,
+  error: null,
+}
+
+const initialState = {
+  KEEP_ETH: { ...wrappedTokenInitialData },
+  TBTC_ETH: { ...wrappedTokenInitialData },
+  KEEP_TBTC: { ...wrappedTokenInitialData },
+}
+
+const liquidityRewardsReducer = (state = initialState, action) => {
+  if (!action.payload) {
+    return state
+  }
+
+  const { liquidityRewardPair, ...restPayload } = action.payload
+
+  switch (action.type) {
+    case `liquidity_rewards/${liquidityRewardPair}_fetch_data_start`:
```

Comment:
Maybe we should keep it all lowercased like other cases in the project? So it would be:

`liquidity_rewards/${liquidityRewardPair.toLowerCase()}_fetch_data_start`

What do you think?

Reply:
I think it's fine and makes sense with uppercase since it's a kind of currency code, like USD PLN etc. Also in exchanges, the currency pairs are uppercased.